src/aux-server/aux-web/aux-auth/site/AuthApp/AuthApp.ts

```diff
@@ -135,39 +140,61 @@ export default class AuthApp extends Vue {
         this.createRecordStudioId = studioId ?? '';
     }
 
+    showError(message: string) {
+        this.showErrorDialog = true;
+        this.errorMessage = message;
+    }
+    closeErrorDialog() {
+        this.showErrorDialog = false;
+        this.errorMessage = '';
+    }
+
     async createStudio() {
-        this.showCreateStudio = false;
-        const comId = authManager.getComIdFromUrl();
-        const result = await authManager.client.createStudio({
-            displayName: this.studioName,
-            ownerStudioComId: comId,
-        });
-        await this.loadStudios();
+        try {
+            this.showCreateStudio = false;
+            const comId = authManager.getComIdFromUrl();
+            const result = await authManager.client.createStudio({
+                displayName: this.studioName,
+                ownerStudioComId: comId,
+            });
+            await this.loadStudios();
+        } catch (error) {
+            this.showError(error.message);
+        }
     }
 
     async createRecord() {
-        this.showCreateRecord = false;
-        let request: Omit<CreateRecordRequest, 'userId'> = {
-            recordName: this.recordName,
-        };
-
-        if (this.createRecordStudioId) {
-            request['studioId'] = this.createRecordStudioId;
-        } else {
-            request['ownerId'] = authManager.userId;
-        }
+        try {
+            this.showCreateRecord = false;
+            let request: Omit<CreateRecordRequest, 'userId'> = {
+                recordName: this.recordName,
+            };
+
+            if (this.createRecordStudioId) {
+                request['studioId'] = this.createRecordStudioId;
+            } else {
+                request['ownerId'] = authManager.userId;
+            }
+
+            const response = await authManager.client.createRecord(request);
 
-        await authManager.client.createRecord(request);
+            if ('errorMessage' in response && response.errorMessage) {
```

Comment:
@TroyceGowdy This should check for if `response.success` equals `false` instead of checking if the `errorMessage` property is defined on the object.